deploy.sh

```diff
@@ -14,16 +14,11 @@
 # limitations under the License.
 set -e
 
-if [ -z "$(git status --porcelain)" ]; then 
-  REVISION="$(git rev-parse --short HEAD)"
-else 
-  echo "Git repo is dirty, continuing anyway..."
-  REVISION="DIRTY_REPO"
-  # echo "Git repo is dirty, cancelling deployment..."
-  # exit 1
-fi
-
-source config.env
+export PROJECT_ID="velociraptor-16p1-test-0"
```

Comment:
question: will we use this project for CI/CD on this repo? Or should this be scrubbed?

Reply:
Eventually this should be scrubbed, but for now it is useful to test outside of the Solutions Console/CFT/Blueprints Controller Framework. Once we get those systems running, we can remove this.